code/gradle.properties

```diff
@@ -24,5 +24,5 @@ mavenRepoDescription=Android Places Extension for Adobe Mobile Marketing
 mavenUploadDryRunFlag=false
 
 # production versions for production build
-mavenCoreVersion=3.0.0-beta.1-SNAPSHOT
+mavenCoreVersion=3.0.0
```

Comment:
The `moduleVersion` is still 3.0.0-beta.1!

Reply:
good catch. thank you!